sharkiq/ayla_api.py

```diff
@@ -48,7 +48,7 @@ def __init__(
         self._app_secret = app_secret
         self.websession = websession
 
-    def ensure_session(self) -> aiohttp.ClientSession:
+    async def ensure_session(self) -> aiohttp.ClientSession:
```

Comment:
A `DeprecationWarning` from `aiohttp` was being issued due to the `aiohttp.ClientSession` object being created outside of an `async` method. Added this and the below `await`s to address this warning.

Reply:
LGTM 👍